src/common/types.ts

```diff
@@ -1,3 +1,35 @@
+import { polygonSchema, multiPolygonSchema } from '@map-colonies/raster-shared';
+import { z } from 'zod';
+
+const featureCollectionPartDataProperties = z.object({
+  sourceId: z.string().optional(),
+  sourceName: z.string(),
+  imagingTimeBeginUTC: z.string(),
+  imagingTimeEndUTC: z.string(),
+  resolutionDegree: z.number(),
+  resolutionMeter: z.number(),
+  sourceResolutionMeter: z.number(),
+  horizontalAccuracyCE90: z.number(),
+  sensors: z.array(z.string()).min(1),
+  countries: z.array(z.string()).optional(),
+  cities: z.array(z.string()).optional(),
+  description: z.string().optional(),
+});
+
+const polygonPartsFeatureSchemaExpanded = z.object({
```

Comment:
i would suggest to rename to `validationFeaureSchema` as it relates to an validation feature structure

Reply:
this can be `validationFeatureCollectionSchema`